### PR TITLE
Fixing code styling for pinot-controller

### PR DESCRIPTION
**Pull request by xiangfu0**
## Description
Fixing code styling for pinot-controller
## Upgrade Notes
Does this PR prevent a zero down-time upgrade? (Assume upgrade order: Controller, Broker, Server, Minion)
* [ ] Yes (Please label as **<code>backward-incompat</code>**, and complete the section below on Release Notes)

Does this PR fix a zero-downtime upgrade introduced earlier?
* [ ] Yes (Please label this as **<code>backward-incompat</code>**, and complete the section below on Release Notes)

Does this PR otherwise need attention when creating release notes? Things to consider:
- New configuration options
- Deprecation of configurations
- Signature changes to public methods/interfaces
- New plugins added or old plugins removed
* [ ] Yes (Please label this PR as **<code>release-notes</code>** and complete the section on Release Notes)

### CODE REVIEW

File: pinot-controller/src/main/java/org/apache/pinot/controller/recommender/rules/impl/KafkaPartitionRule.java [hunk 1]
```diff
@@ -35,29 +35,27 @@
...
   @Override
   public void run() {
     String tableType = _input.getTableType();
-    if ((tableType.equalsIgnoreCase(HYBRID) || tableType
-        .equalsIgnoreCase(REALTIME))) { //The table is real-time or hybrid
-      if (_input.getNumKafkaPartitions()
-          == DEFAULT_NUM_KAFKA_PARTITIONS)  // Recommend NumKafkaPartitions if it is not given
-      {
+    if ((tableType.equalsIgnoreCase(HYBRID) || tableType.equalsIgnoreCase(REALTIME))) {
+      //The table is real-time or hybrid
```
Comment (jackjlli) [inline pinot-controller/src/main/java/org/apache/pinot/controller/recommender/rules/impl/KafkaPartitionRule.java:50]: nit: put a space after `//`.

File: pinot-controller/src/main/java/org/apache/pinot/controller/recommender/rules/impl/KafkaPartitionRule.java [hunk 2]
```diff
@@ -35,29 +35,27 @@
...
+      if (_input.getNumKafkaPartitions() == DEFAULT_NUM_KAFKA_PARTITIONS) {
+        // Recommend NumKafkaPartitions if it is not given
         LOGGER.info("Recommending kafka partition configurations");
         LOGGER.info("*No kafka partition number found, recommending kafka partition number");
         _output.getPartitionConfig().setNumKafkaPartitions((int) Math
-            .ceil((double) _input.getNumMessagesPerSecInKafkaTopic() / _params.KAFKA_NUM_MESSAGES_PER_SEC_PER_PARTITION));
+            .ceil((double) _input.getNumMessagesPerSecInKafkaTopic() / _params._kafkaNumMessagesPerSecPerPartition));
         //Divide the messages/sec (total aggregate in the topic) by 250 to get an optimal value of the number of kafka partitions.
```
Comment (jackjlli) [inline pinot-controller/src/main/java/org/apache/pinot/controller/recommender/rules/impl/KafkaPartitionRule.java:57]: nit: put a space after `//`.

File: pinot-controller/src/main/java/org/apache/pinot/controller/recommender/rules/io/params/BloomFilterRuleParams.java [hunk 3]
```diff
@@ -27,26 +27,26 @@
...
+    return _thresholdMaxCardinalityBloomfilter;
   }
 
   @JsonSetter(value = "THRESHOLD_MAX_CARDINALITY_BLOOMFILTER", nulls = Nulls.SKIP)
-  public void setTHRESHOLD_MAX_CARDINALITY_BLOOMFILTER(Long THRESHOLD_MAX_CARDINALITY_BLOOMFILTER) {
-    this.THRESHOLD_MAX_CARDINALITY_BLOOMFILTER = THRESHOLD_MAX_CARDINALITY_BLOOMFILTER;
+  public void setThresholdMaxCardinalityBloomfilter(Long thresholdMaxCardinalityBloomfilter) {
+    _thresholdMaxCardinalityBloomfilter = thresholdMaxCardinalityBloomfilter;
   }
 }
```
Comment (jackjlli) [inline pinot-controller/src/main/java/org/apache/pinot/controller/recommender/rules/io/params/BloomFilterRuleParams.java:52]: Missing an empty line at the end.

File: pinot-controller/src/main/java/org/apache/pinot/controller/recommender/rules/io/params/InvertedSortedIndexJointRuleParams.java [hunk 4]
```diff
@@ -31,123 +32,122 @@
...
+    return _percentSelectForIsnull;
   }
 
   @JsonSetter(value = "PERCENT_SELECT_FOR_ISNULL", nulls = Nulls.SKIP)
-  public void setPERCENT_SELECT_FOR_ISNULL(Double PERCENT_SELECT_FOR_ISNULL) {
-    this.PERCENT_SELECT_FOR_ISNULL = PERCENT_SELECT_FOR_ISNULL;
+  public void setPercentSelectForIsnull(Double percentSelectForIsnull) {
+    _percentSelectForIsnull = percentSelectForIsnull;
   }
 }
```
Comment (jackjlli) [inline pinot-controller/src/main/java/org/apache/pinot/controller/recommender/rules/io/params/InvertedSortedIndexJointRuleParams.java:153]: Missing an empty line at the end.

File: pinot-controller/src/main/java/org/apache/pinot/controller/recommender/rules/io/params/NoDictionaryOnHeapDictionaryJointRuleParams.java [hunk 5]
```diff
@@ -23,97 +23,97 @@
...
+    return _thresholdMinFilterFreqDictionary;
   }
 
   @JsonSetter(value = "THRESHOLD_MIN_FILTER_FREQ_DICTIONARY", nulls = Nulls.SKIP)
-  public void setTHRESHOLD_MIN_FILTER_FREQ_DICTIONARY(Double THRESHOLD_MIN_FILTER_FREQ_DICTIONARY) {
-    this.THRESHOLD_MIN_FILTER_FREQ_DICTIONARY = THRESHOLD_MIN_FILTER_FREQ_DICTIONARY;
+  public void setThresholdMinFilterFreqDictionary(Double thresholdMinFilterFreqDictionary) {
+    _thresholdMinFilterFreqDictionary = thresholdMinFilterFreqDictionary;
   }
 }
```
Comment (jackjlli) [inline pinot-controller/src/main/java/org/apache/pinot/controller/recommender/rules/io/params/NoDictionaryOnHeapDictionaryJointRuleParams.java:119]: Missing an empty line at the end.

**Review by jackjlli — APPROVED** [review]
Minor but LGTM

File: pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableSegmentConfigs.java [hunk 6]
```diff
@@ -49,30 +49,31 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(PinotTableSegmentConfigs.class);
 
   @Inject
-  PinotHelixResourceManager pinotHelixResourceManager;
+  PinotHelixResourceManager _pinotHelixResourceManager;
   @Inject
-  ControllerMetrics metrics;
+  ControllerMetrics _metrics;
```
[the remark this answers is not in the document]
Comment (xiangfu0) [reply pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableSegmentConfigs.java]: deleted.

**Pull request merged**